### PR TITLE
erlang/elixir getting started: add section using otlp exporter

### PR DESCRIPTION
**Pull request by tsloughter**
Edit (@chalin):

Preview: https://deploy-preview-1167--opentelemetry.netlify.app/docs/instrumentation/erlang/getting-started/

### CODE REVIEW

File: content/en/docs/instrumentation/erlang/getting-started.md [hunk 1]
```diff
@@ -267,3 +264,118 @@ iex(2)>
...
+
+The [Collector](/docs/collector/) provides a vendor
+agnostic way to receive, process and export telemetry data. The package
+[opentelemetry_exporter](https://hex.pm/packages/opentelemetry_exporter)
+provides support for both exporting over both HTTP (the default) and GRPC to the
+collector which can then export Spans to a self-hosted service like Zipkin or
+Jaeger, as well as commercial services. See the [OpenTelemetry Collector's
+contrib
+repo](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter)
+for a full list of available exporters in the Collector.
```
Comment (chalin) [inline content/en/docs/instrumentation/erlang/getting-started.md:278]: Two things here -- consider the following:

- Redirect the reader to the registry at https://opentelemetry.io/registry/?component=exporter instead.
- Rephrase as:
  > For a full list of available exporters, see <link>.

File: content/en/docs/instrumentation/erlang/getting-started.md [hunk 2]
```diff
@@ -267,3 +264,118 @@ iex(2)>
 {{< /tab >}}
 
 {{< /tabs >}}
+
+## Exporting to the OpenTelemetry Collector
+
+The [Collector](/docs/collector/) provides a vendor
+agnostic way to receive, process and export telemetry data. The package
+[opentelemetry_exporter](https://hex.pm/packages/opentelemetry_exporter)
+provides support for both exporting over both HTTP (the default) and GRPC to the
```
Comment (chalin) [inline content/en/docs/instrumentation/erlang/getting-started.md:273]: Here and elsewhere: GRPC -> gRPC

File: content/en/docs/instrumentation/erlang/getting-started.md [hunk 3]
```diff
@@ -267,3 +264,118 @@ iex(2)>
...
+{relx, [{release, {my_instrumented_release, "0.1.0"},
+         [opentelemetry_exporter,
+	      {opentelemetry, temporary},
+          my_instrumented_app]},
+
+       ...]}.
+{{< /tab >}}
+
+{{< tab >}}
+# mix.exs
```
Comment (chalin) [inline content/en/docs/instrumentation/erlang/getting-started.md:331]: Do want two leading `#`s here?

**Review by chalin — APPROVED** [review]
See inline comments. Overall LGTM from a tech docs perspective. I have more copyedits to make, but I can do those after this PR gets merged.

**Comment by chalin** [comment]
@austinlparker @cartermp - any comments on this PR?

File: content/en/docs/instrumentation/erlang/getting-started.md (hunk 1 above)
Comment (tsloughter) [reply content/en/docs/instrumentation/erlang/getting-started.md]: Good point about the registry. I've updated the phrasing and the link.

File: content/en/docs/instrumentation/erlang/getting-started.md (hunk 2 above)
Comment (tsloughter) [reply content/en/docs/instrumentation/erlang/getting-started.md]: Done.

File: content/en/docs/instrumentation/erlang/getting-started.md (hunk 3 above)
Comment (tsloughter) [reply content/en/docs/instrumentation/erlang/getting-started.md:330]: Na, Elixir convention is just one `#`, while Erlang convention for comments on a line of its own is two `%%`.

Comment (chalin) [reply content/en/docs/instrumentation/erlang/getting-started.md:330]: Ok, I was just wondering about the inconsistency. `##` is used on these lines (of this page):

- L133, at https://github.com/open-telemetry/opentelemetry.io/pull/1167/files#diff-bfa43494e057aca10a8d8fd612bd226d5c546f3b9aaffeb551a62fdf43ed7f22R133
- L368, at https://github.com/open-telemetry/opentelemetry.io/pull/1167/files#diff-bfa43494e057aca10a8d8fd612bd226d5c546f3b9aaffeb551a62fdf43ed7f22R368

Should those two instance be using a single `#` too?

**Review by chalin — COMMENTED** [review]
Thanks for the updates. See one last followup comment about the consistency of using `#` vs `##` in this page.

File: content/en/docs/instrumentation/erlang/getting-started.md (hunk 3 above)
Comment (tsloughter) [reply content/en/docs/instrumentation/erlang/getting-started.md:330]: Ah! Yea, those should be a single `#` I'll push another commit fixing that up.

**Review by chalin — APPROVED** [review]

**Pull request merged**